### PR TITLE
Nominate Tatsuya Sato as a maintainer

### PR DESCRIPTION
**Pull request by denyeart**
Tatsuya Sato has been contributing to Fabric for several years. In the last 6 months Tatsuya has increased contributions, becoming one of the most active contributors.

Tatsuya has contributed new chaincode lifecycle functions that required updating protos across Fabric repositories.
He has fixed issues with cert expiration warnings.

Beyond normal code contributions, Tatsuya has also been doing general  maintenance pull requests, for example maintaining the installation script, cleaning up Fabric docs,
and maintaining the fabric-samples repository.
He has also translated documentation into Japanese, and been active in pull request reviews.

Tatsuya has demonstrated knowledge and added value across a wide range of areas spanning new features and general maintenance. In short, Tatsuya has already been maintaining Fabric. I would welcome Tatsuya's help as an official maintainer.

### CODE REVIEW

**Comment by satota2** [comment]
@denyeart @hyperledger/fabric-core-maintainers 
Thank you for the nomination.

I am deeply honored to be considered for the role of a maintainer. I look forward to the possibility of contributing even more to the project as a maintainer, with everyone's approval.

Thank you once again.

**Comment by takakir** [comment]
@satota2 HLF users including myself are very lucky to have such distinguished individual as the HLF maintainer. 
Yoroshiku onegai shimasu!

**Review by andrew-coleman — APPROVED** [review]
Welcome @satota2

**Review by yacovm — APPROVED** [review]

**Review by cendhu — APPROVED** [review]

**Review by tock-ibm — APPROVED** [review]

**Comment by denyeart** [comment]
We have received approval from majority of existing maintainers. I'll go ahead and merge, congratulations and welcome @satota2 ! I'll grant you permissions on fabric-core-maintainers.

**Comment by satota2** [comment]
Thank you to the existing maintainers for approving me as a maintainer.
I've confirmed my account now has fabric-core-maintainers permissions.

I am looking forward to making meaningful contributions to Fabric in my capacity as a maintainer.
Once again, I am deeply grateful for this opportunity.

**Pull request merged**
